src/libstd/keyword_docs.rs

```diff
@@ -608,6 +608,62 @@ mod in_keyword { }
 /// [Reference]: ../reference/statements.html#let-statements
 mod let_keyword { }
 
+#[doc(keyword = "while")]
+//
+/// Loop while a condition is upheld.
+///
+/// A `while` expression is used for predicate loops. The `while` expression runs the conditional
+/// expression before running the loop body, then runs the loop body if the conditional
+/// expression evaluates to `true`, or exits the loop otherwise.
+///
+/// ```rust
```

Comment:
There is no need to explicit specify rust here and other places.

Reply:
But it's also not a problem if you do; I tend to do it for clarity.

Reply:
It is also used in other places and we can leave it to be clear.